experimental/LieAlgebras/src/LieAlgebraModule.jl

```diff
@@ -1242,3 +1228,154 @@ function tensor_power(V::LieAlgebraModule{C}, k::Int) where {C<:FieldElem}
   )
   return T
 end
+
+###############################################################################
+#
+#   Simple modules (via highest weight) of semisimple Lie algebras
+#
+###############################################################################
+
+# TODO: check semisimplicity check once that is available
+
+function is_dominant_weight(hw::Vector{Int})
+  return all(>=(0), hw)
+end
+
+@doc raw"""
+    simple_module(L::LieAlgebra{C}, hw::Vector{Int}) -> LieAlgebraModule{C}
+
+Construct the simple module of the Lie algebra `L` with highest weight `hw`.
+The actual construction is done in GAP.
+"""
+function simple_module(L::LieAlgebra, hw::Vector{Int})
+  @req is_dominant_weight(hw) "Not a dominant weight."
+  struct_consts = lie_algebra_simple_module_struct_consts_gap(L, hw)
+  dimV = size(struct_consts, 2)
+  V = abstract_module(L, dimV, struct_consts; check=false)
+  set_attribute!(V, :is_simple => true, :highest_weight => hw)
```

Comment:
This line currently conveys no meaning at all, as there is no `is_simple` method for modules yet. So this will be changed again in the future either way.

Reply:
Can you remove the sentence mentioning "GAP" from the docstring while you are at it?

Reply:
> This line currently conveys no meaning at all, as there is no `is_simple` method for modules yet. So this will be changed again in the future either way.

Then just remove the line ?